phi/tools/scrapegraph_tools.py

```diff
@@ -0,0 +1,68 @@
+import json
+from typing import Optional
+from phi.tools import Toolkit
+
+try:
+    from scrapegraph_py import Client
+except ImportError:
+    raise ImportError("`scrapegraph-py` not installed. Please install using `pip install scrapegraph-py`")
+
+
+class ScrapeGraphTools(Toolkit):
+    def __init__(
+        self,
+        api_key: Optional[str] = None,
+        smartscraper: bool = True,
+        markdownify: bool = False,
+    ):
+        super().__init__(name="scrapegraph_tools")
+
+        self.api_key: Optional[str] = api_key
```

Comment:
add `api_key or os.getenv("SGAI_API_KEY")`

Reply:
Thanks for the feedback! Added it